src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -64,6 +66,39 @@ internal struct NameDeclarationInfo
                 return default;
             }
 
+            private static bool IsTupleElementDeclaration(
```

Comment:
Nit: Consider renaming this method to `IsTupleTypeElement` and the following one `IsTupleLiteralElement`.

Reply:
Done cf1a9739.